pickem/pickem_homepage/views.py

```
@@ -4041,6 +4076,248 @@ def serialize_comment(comment):
+                'success': False,
+                'error': 'You do not have permission to delete this post'
+            }, status=403)
+
+        post.is_active = False
+        post.save(update_fields=['is_active'])
+
+        return JsonResponse({'success': True})
+
+    except Http404:
+        return message_board_not_found()
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)